### PR TITLE
Privacy 

### PR DESCRIPTION
**Pull request by EmmaFrith**
add privacy policy

### CODE REVIEW

File: app/views/privacy-policy.html [hunk 1]
```diff
@@ -2,15 +2,125 @@
...
+          <li><a class="govuk-link" href="#How-we-use-third-parties-to-process-your-data">How we use third parties to process your data</a></li>
+          <li><a class="govuk-link" href="#How-long-we-keep-your-personal-data">How long we keep your personal data</a></li>
+          <li><a class="govuk-link" href="#Raise-a-concern">Raise a concern</a></li>
+          <li><a class="govuk-link" href="#Keep-up-to-date">Keep up to date</a></li>
+        </ul>
+      </div>
+    </div>
+
+    <section id="What-personal-data-we-collect-and-why">
+    <h2 class="govuk-heading-m">What personal data we collect and why</h2>
```
Comment (jesseyuen) [inline app/views/privacy-policy.html]: All the markup within each of the `<section>` elements should be indented as best practice.

**Review by jesseyuen — APPROVED** [review]

File: app/views/privacy-policy.html (hunk 1 above)
Comment (EmmaFrith) [reply app/views/privacy-policy.html]: thanks @jesseyuen - have indented the content within the sections now!

File: app/views/privacy-policy.html [hunk 2]
```diff
@@ -2,15 +2,127 @@
...
 
 {% block content %}
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
-    <h1 class="govuk-heading-l">{{pageHeading}}</h1>
-    <p class="govuk-body">Content goes here</p>
-  </div>
-</div>
+    <span class="govuk-caption-xl">Privacy policy</span>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]:
```suggestion
    <span class="govuk-caption-l">Privacy policy</span>
```

File: app/views/privacy-policy.html [hunk 3]
```diff
@@ -2,15 +2,127 @@
...
+
+    <p class="govuk-body">This service allows you to share trainee data with the Department for Education (DfE).</p>
+
+    <p class="govuk-body">However, by using the service, you’ll also be sharing some information about you.</p>
+
+    <div class="govuk-inset-text">This privacy policy is about your personal data, as opposed to your trainee data.</div>
+
+    <p class="govuk-body">With regard to your trainee data, remember to meet your obligations under the General Data Protection Legislation as a data controller for their information.</p>
+
+    <p class="govuk-body govuk-!-margin-bottom-7">You can <a href="https://ico.org.uk/for-organisations/" class="govuk-link">read about your data protection obligations by visiting the Information Commissioner's Office (ICO) website</a>.</p>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]:
```suggestion
    <p class="govuk-body govuk-!-margin-bottom-7">You can <a href="https://ico.org.uk/for-organisations/" class="govuk-link">read about your data protection obligations by visiting the Information Commissioner’s Office (ICO) website</a>.</p>
```

File: app/views/privacy-policy.html [hunk 4]
```diff
@@ -2,15 +2,127 @@
...
+        <ul class="govuk-list govuk-list--dash govuk-!-margin-bottom-8">
+          <li><a class="govuk-link" href="#What-personal-data-we-collect-and-why">What personal data we collect and why</a></li>
+          <li><a class="govuk-link" href="#Our-lawful-basis-for-processing-your-personal-data">Our lawful basis for processing your personal data</a></li>
+          <li><a class="govuk-link" href="#How-we-use-third-parties-to-process-your-data">How we use third parties to process your data</a></li>
+          <li><a class="govuk-link" href="#How-long-we-keep-your-personal-data">How long we keep your personal data</a></li>
+          <li><a class="govuk-link" href="#Raise-a-concern">Raise a concern</a></li>
+          <li><a class="govuk-link" href="#Keep-up-to-date">Keep up to date</a></li>
+        </ul>
+      </div>
+    </div>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: I don't think these two divs are needed - the entire section is already in a column

File: app/views/privacy-policy.html [hunk 5]
```diff
@@ -2,15 +2,127 @@
...
+      <p class="govuk-body govuk-!-margin-bottom-7">If you take part in user research, you may choose to share additional information about you.</p>
+
+    <section id="Our-lawful-basis-for-processing-your-personal-data">
+      <h2 class="govuk-heading-m">Our lawful basis for processing your personal data</h2>
+
+      <p class="govuk-body">Running and improving the service is a public task, as defined in data protection law. However, we ask for your consent to take part in user research.</p>
+
+      <p class="govuk-body govuk-!-margin-bottom-7"><a href="https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/lawful-basis-for-processing/" class="govuk-link">Read about consent and public task on the Information Commissioner's Office (ICO) website</a>.</p>
+
+    <section id="How-we-use-third-parties-to-process-your-data">
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: I'm not sure a `<section>` is needed here. If the main purpose is to make the anchor links work, you can add the `id="foo"` to the h2s.

File: app/views/privacy-policy.html [hunk 6]
```diff
@@ -2,15 +2,127 @@
...
+        <li>send important updates about the service</li>
+        <li>get in touch with you about any queries you raise</li>
+        <li>ask if you would like to take part in user research</li>
+        <li>record a timeline of your activity so that you can review trainee progress</li>
+        <li>create log files of your activity and any error messages you come across</li>
+      </ul>
+
+      <p class="govuk-body govuk-!-margin-bottom-7">If you take part in user research, you may choose to share additional information about you.</p>
+
+    <section id="Our-lawful-basis-for-processing-your-personal-data">
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: I'm not sure a `<section>` is needed here. If the main purpose is to make the anchor links work, you can add the `id="foo"` to the h2s.

File: app/views/privacy-policy.html [hunk 7]
```diff
@@ -2,15 +2,127 @@
...
+          <li><a class="govuk-link" href="#Our-lawful-basis-for-processing-your-personal-data">Our lawful basis for processing your personal data</a></li>
+          <li><a class="govuk-link" href="#How-we-use-third-parties-to-process-your-data">How we use third parties to process your data</a></li>
+          <li><a class="govuk-link" href="#How-long-we-keep-your-personal-data">How long we keep your personal data</a></li>
+          <li><a class="govuk-link" href="#Raise-a-concern">Raise a concern</a></li>
+          <li><a class="govuk-link" href="#Keep-up-to-date">Keep up to date</a></li>
+        </ul>
+      </div>
+    </div>
+
+    <section id="What-personal-data-we-collect-and-why">
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: I'm not sure a `<section>` is needed here. If the main purpose is to make the anchor links work, you can add the `id="foo"` to the h2s.

File: app/views/privacy-policy.html [hunk 8]
```diff
@@ -2,15 +2,127 @@
...
+
+      <p class="govuk-body">We use Zendesk to manage and respond to your queries.</p>
+
+      <p class="govuk-body"><a href="https://www.zendesk.co.uk/company/privacy-and-data-protection/" class="govuk-link">Visit the Zendesk website to find out how they use and look after your data ("service data")</a>.</p>
+
+      <h3 class="govuk-heading-s">Website hosting services</h3>
+
+      <p class="govuk-body">We use GOV.UK PaaS to enable us to run the service.</p>
+
+      <p class="govuk-body"><a href="https://www.cloud.service.gov.uk/privacy-notice/#what-data-we-collect-from-end-users" class="govuk-link">Visit the GOV.UK PaaS website to find out how they use and look after your data ("end user data")</a>.</p>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: wonder about the length of this link and whether the `("end user data")` bit should be outside of it.

File: app/views/privacy-policy.html [hunk 9]
```diff
@@ -2,15 +2,127 @@
...
+
+      <h3 class="govuk-heading-s">Error and activity logging services</h3>
+
+      <p class="govuk-body">We need to keep some records so that we can identify and fix problems with the service. We use Sentry to monitor errors and Logit to keep activity logs.</p>
+
+      <p class="govuk-body">We do not identify you from these records unless needed - for example, to investigate an issue you raise</p>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]:
```suggestion
      <p class="govuk-body">We do not identify you from these records unless needed - for example, to investigate an issue you raise.</p>
```

File: app/views/privacy-policy.html [hunk 10]
```diff
@@ -2,15 +2,127 @@
...
+
+      <p class="govuk-body"><a href="https://sentry.io/security/" class="govuk-link">Visit Sentry's website to read about their security practices</a>.</p>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]:
```suggestion
      <p class="govuk-body"><a href="https://sentry.io/security/" class="govuk-link">Visit Sentry’s website to read about their security practices</a>.</p>
```

File: app/views/privacy-policy.html [hunk 11]
```diff
@@ -2,15 +2,127 @@
...
+
+      <p class="govuk-body"><a href="https://logit.io/security" class="govuk-link">Visit Logit's website to read about their security practices</a>.</p>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]:
```suggestion
      <p class="govuk-body"><a href="https://logit.io/security" class="govuk-link">Visit Logit’s website to read about their security practices</a>.</p>
```

File: app/views/privacy-policy.html [hunk 12]
```diff
@@ -2,15 +2,127 @@
...
+
+      <h3 class="govuk-heading-s">User research tools</h3>
+
+      <p class="govuk-body">If you agree to take part in user research, we’ll ask if you consent to use Lookback to carry out and record a video session with you.</p>
+
+      <p class="govuk-body govuk-!-margin-bottom-7"><a href="https://lookback.io/privacy/" class="govuk-link">Visit Lookback's website to read about how they use your data (as a "participant")</a>.</p>
+
+    <section id="How-long-we-keep-your-personal-data">
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: Remove section

File: app/views/privacy-policy.html [hunk 13]
```diff
@@ -2,15 +2,127 @@
...
+      <h2 class="govuk-heading-m">How long we keep your personal data</h2>
+
+      <p class="govuk-body">We keep your personal data for as long as you use the service.</p>
+
+      <p class="govuk-body">We may also keep it for some time after you stop using the service. This is so that we can analyse any feedback you provided to improve the service.</p>
+
+      <p class="govuk-body govuk-!-margin-bottom-7">We’ll keep your data for no longer than 7 years after you stop using the service.</p>
+
+    <section id="Raise-a-concern">
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: remove section

File: app/views/privacy-policy.html [hunk 14]
```diff
@@ -2,15 +2,127 @@
...
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <ul class="govuk-list govuk-list--dash govuk-!-margin-bottom-8">
+          <li><a class="govuk-link" href="#What-personal-data-we-collect-and-why">What personal data we collect and why</a></li>
+          <li><a class="govuk-link" href="#Our-lawful-basis-for-processing-your-personal-data">Our lawful basis for processing your personal data</a></li>
+          <li><a class="govuk-link" href="#How-we-use-third-parties-to-process-your-data">How we use third parties to process your data</a></li>
+          <li><a class="govuk-link" href="#How-long-we-keep-your-personal-data">How long we keep your personal data</a></li>
+          <li><a class="govuk-link" href="#Raise-a-concern">Raise a concern</a></li>
+          <li><a class="govuk-link" href="#Keep-up-to-date">Keep up to date</a></li>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: Would recommend making all of these anchors lowercase only.

File: app/views/privacy-policy.html [hunk 15]
```diff
@@ -2,15 +2,127 @@
...
+
+      <p class="govuk-body">Contact the Register trainee teachers team at <a class="govuk-link" href="mailto:registerateacher@digital.education.gov.uk?subject=My%20personal%20data%20">registerateacher@digital.education.gov.uk</a> if you have any concerns about your personal data.</p>
+
+      <p class="govuk-body">You can also <a href="https://form.education.gov.uk/en/AchieveForms/?form_uri=sandbox-publish://AF-Process-f1453496-7d8a-463f-9f33-1da2ac47ed76/AF-Stage-1e64d4cc-25fb-499a-a8d7-74e98203ac00/definition.json&redirectlink=%2Fen&cancelRedirectLink=%2Fen" class="govuk-link">get in touch with the Department for Education’s data protection officer through the online contact form</a>.</p>
+
+      <p class="govuk-body govuk-!-margin-bottom-7"><a href="https://www.gov.uk/government/organisations/department-for-education/about/personal-information-charter" class="govuk-link">Refer to the Department for Education's personal information charter to read more about the standards you can expect from us when we look after your personal data</a>.</p>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: Are these `govuk-!-margin-bottom-7` intentional on each paragraph?

Comment (edwardhorsford) [inline app/views/privacy-policy.html]:
```suggestion
      <p class="govuk-body govuk-!-margin-bottom-7"><a href="https://www.gov.uk/government/organisations/department-for-education/about/personal-information-charter" class="govuk-link">Refer to the Department for Education’s personal information charter to read more about the standards you can expect from us when we look after your personal data</a>.</p>
```

File: app/views/privacy-policy.html [hunk 16]
```diff
@@ -2,15 +2,127 @@
...
+
+    <section id="Keep-up-to-date">
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]: Remove section

File: app/views/privacy-policy.html [hunk 17]
```diff
@@ -2,15 +2,127 @@
...
+      <h2 class="govuk-heading-m">Keep up to date</h2>
+
+      <p class="govuk-body">We periodically review this policy. We'll notify you if we change how we use your personal data.</p>
```
Comment (edwardhorsford) [inline app/views/privacy-policy.html]:
```suggestion
      <p class="govuk-body">We periodically review this policy. We’ll notify you if we change how we use your personal data.</p>
```

**Review by edwardhorsford — CHANGES_REQUESTED** [review]

**Pull request merged**